bcda/cclf/importer.go

```diff
@@ -14,7 +13,7 @@ import (
 )
 
 type importer interface {
-	do(ctx context.Context, tx *sql.Tx, fileID uint, b []byte) error
+	do(ctx context.Context, tx *sql.Tx, cclfBeneficiary models.CCLFBeneficiary) error
```

Comment:
To keep this generic can we change the argument to an interface{}? Then you can use a type assertion to verify that we passed a CCLFBeneficiary.

It would look something like this:
```
func (cclfImporter *cclf8Importer) do(ctx context.Context, tx *sql.Tx, data interface{}) {
     bene, ok := data.(models.CCLFBeneficiary)
     if !ok {
          return fmt.Errrof("invalid type sent, expected models.CCLFBeneficiary)
     }
     ....
}
```